products/oracle-linux.md

```diff
@@ -26,6 +26,13 @@ identifiers:
 -   purl: pkg:docker/library/oraclelinux
 
 releases:
+-   releaseCycle: "10"
+    releaseDate: 2025-06-26
+    eol: 2035-06-30
+    eoes: 2038-06-30
```

Comment:
```suggestion
    eol: false
    eoes: false
```

Where can I confirm those ? 10 is not listed on https://www.oracle.com/a/ocom/docs/elsp-lifetime-069338.pdf.

Reply:
Those were estimated based on 10 years support which we were mentioning :
```
**Basic Support**: Available for 10 years for versions 5–9 from date of release. Includes access
  to patches, fixes, security patches and security alerts.
  ```

Reply:
This explicitly states `for versions 5–9`.